containers/ecr-viewer/src/app/services/fhirDataService.ts

```diff
@@ -42,17 +44,21 @@ export const isSuccessResponse = (resp: Response): resp is SuccessResponse =>
  * @param ecr_id The id of the ecr to fetch
  * @returns NextResponse with the ecr or error data
  */
-export async function getFhirData(ecr_id: string | null) {
-  if (process.env.SOURCE === S3_SOURCE) {
-    return await getS3(ecr_id);
-  } else if (process.env.SOURCE === AZURE_SOURCE) {
-    return await getAzure(ecr_id);
-  } else if (process.env.SOURCE === GCP_SOURCE) {
-    return await getGcp(ecr_id);
-  } else {
-    return { payload: { message: "Invalid source" }, status: 500 };
-  }
-}
+export const getFhirData = audit(
+  "ecr",
+  "view",
+  async ({ ecr_id }: { ecr_id: string | null }) => {
```

Comment:
```suggestion
  async ({ eicr_id }: { eicr_id: string | null }) => {
```

Could we switch this to `eicr_id` so it matches the other audit function param names?